tools/cfn2ts/lib/codegen.ts

```diff
@@ -138,13 +138,24 @@ export default class CodeGenerator {
    *
    * Return a mapping of { originalName -> newName }.
    */
-  private emitPropsTypeProperties(resource: genspec.CodeName, propertiesSpec: { [name: string]: schema.Property }): Dictionary<string> {
+  private emitPropsTypeProperties(
+    resource: genspec.CodeName,
+    propertiesSpec: { [name: string]: schema.Property },
+    container: Container): Dictionary<string> {
+
     const propertyMap: Dictionary<string> = {};
 
     Object.keys(propertiesSpec).sort(propertyComparator).forEach(propName => {
+      this.code.line();
       const propSpec = propertiesSpec[propName];
       const additionalDocs = resource.specName!.relativeName(propName).fqn;
-      const newName = this.emitProperty(resource, propName, propSpec, quoteCode(additionalDocs));
+      const newName = this.emitProperty({
```

Comment:
I'm not sure why I left this an object, I'm going to move back to normal args here.

Reply:
I like the props. Definitely more readable.

Reply:
Please resolve all conversations that you attended to